performance/config/budget.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
De config bevat default values? We moeten 'onze' values nog bepalen?

Reply:
Yes, wanneer alles up-and-running is moeten deze waardes nog getweaked worden.